### ISSUE TITLE
Backend: User data dump (& import)

### ISSUE THREAD

**Issue by nogweii** [samuelclay/NewsBlur#86]
I would really like to see a way to export all of a user's data from NewsBlur. One case is basically backing up everything - shared posts, blurblog, feed list & intelligence settings. Another use case would be greatly simplifying the transition from hosted NewsBlur to self-hosted/private NewsBlur instances.

**Comment by samuelclay** [comment]
The API is perfect for this. www.newsblur.com/api

**Comment by Limero** [comment]
Any chance this can be reopened? A one button "full export" feature in the web interface for premium users would be a huge quality of life improvement. I don't want to risk losing the work spent configuring Intelligence Trainer. This feature missing is a big reason to self-host NewsBlur where backups are simple to do, instead of using the officially hosted version.

**Comment by samuelclay** [comment]
You're right, a better one-click that downloads all the major tables in json format would be great. This should integrate saved stories as well as trainer data, feeds, and probably preferences. Import would be a necessary component.
